src/TextifyBot.py

```diff
@@ -96,6 +96,8 @@ def improveImage(imagePath):
 
 def transcribeImages(imagesToDL): # download and transcribe a list of image URLs
     transcribedText = []
+    transcriptionSpacing = "\n  ----------------------------  \n"
```

Comment:
```suggestion
    transcriptionSpacing = "\n---\n"
```

In Markdown, three or more `-`'s are converted into a horizontal rule, so no need to have any more than three.